### PR TITLE
[Work in progress] Speaker sharing images

### PR DESCRIPTION
**Pull request by adrianmoisey**
This is mostly so I can test with a public URL.

### CODE REVIEW

**Comment by adrianmoisey** [comment]
Example:

image


I think this solves https://github.com/devopsdays/devopsdays-web/issues/6538

File: content/events/2019-minneapolis/speakers/chelsey-mckinney.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 Title = "Chelsey McKinney"
 Twitter = "techchelsey"
 image = "chelsey-mckinney.png"
+sharing_url = "chelsey-mckinney-card.png"
```
Comment (adrianmoisey) [inline content/events/2019-minneapolis/speakers/chelsey-mckinney.md]: Should this be renamed to `sharing_image` ?

**Comment by adrianmoisey** [comment]
If everyone is happy that this is the right solution (cc @mattstratton), I'll remove the 'demo' and update the docs.

**Comment by MikeRosTX** [comment]
We have these cards created every year and this Speaker sharing image would be an awesome feature added to the them. ❤️ 👍 

image

File: content/events/2019-minneapolis/speakers/chelsey-mckinney.md (hunk 1 above)
Comment (MikeRosTX) [reply content/events/2019-minneapolis/speakers/chelsey-mckinney.md]: You know how we have a Twitter,, LinkedIn, GitHub, Website  symbols under the speaker's picture on the Speaker's page?  Can you add a sharing symbol like the following when an image is used?

https://img.favpng.com/6/1/19/share-icon-computer-icons-android-file-sharing-png-favpng-fEKtmyz51BA6JcxbEZK7rJyiX_t.jpg

Comment (mattstratton) [reply content/events/2019-minneapolis/speakers/chelsey-mckinney.md]: I think adding a sharing CTA on other pages is not a bad idea, but i wouldn’t put it there.

**Comment by adrianmoisey** [comment]
@MikeRosTX Using your card as an example:
image

Looks awesome!

File: content/events/2019-minneapolis/speakers/chelsey-mckinney.md (hunk 1 above)
Comment (MikeRosTX) [reply content/events/2019-minneapolis/speakers/chelsey-mckinney.md]: Where would you put it? Because it's supposed to be by speaker. Is it not?

Comment (adrianmoisey) [reply content/events/2019-minneapolis/speakers/chelsey-mckinney.md]: Would that sharing symbol direct link to the `sharing_image`?

**Comment by MikeRosTX** [comment]
@mattstratton in the REFERENCE.md under the Speaker_page_fields. 😉 

https://github.com/devopsdays/devopsdays-web/blob/master/themes/devopsdays-theme/REFERENCE.md#speaker-page-fields

File: content/events/2019-minneapolis/speakers/chelsey-mckinney.md (hunk 1 above)
Comment (mattstratton) [reply content/events/2019-minneapolis/speakers/chelsey-mckinney.md]: The other actions are links to the speaker’s stuff. 

A sharing button is for sharing the whole page. 

Totally different CTA’s and we have already seen how it confuses people having the sharing buttons so close to the follow buttons on the event welcome pages.

**Comment by MikeRosTX** [comment]
@adrianmoisey you have one demo on **_{speaker_name}.md_** and the other one on **_{speaker_program}.md_**.  Which one it be final against or both?

**Comment by mattstratton** [comment]
> you have one demo on {speaker_name}.md and the other one on {speaker_program}.md. Which one it be final against or both?

This works on ANY `*.md` file in your site. So if you wanted an image for your "Location" page, for example, you could set that too!

This is NOT speaker/talk specific. This is a global change which can be used on any page in your event's site.

**Comment by mattstratton** [comment]
>in the REFERENCE.md under the Speaker_page_fields.
Huh?

**Comment by mattstratton** [comment]
@adrianmoisey 	I recommend changing the title on this PR to say "add sharing images on any page" since the current title implies this is for speaker/talk pages only :)

**Comment by adrianmoisey** [comment]
> @adrianmoisey you have one demo on **_{speaker_name}.md_** and the other one on **_{speaker_program}.md_**. Which one it be final against or both?

Just adding more to what Matty already said, the `seo.html` file has this in it:
```{{- if or (eq .Type "welcome") (eq .Type "event") (eq .Type "speaker") (eq .Type "speakers") (eq .Type "talk") (eq .Type "program") -}}
```

I've added the `sharing_image` inside that code block.
This means that it's a general solution for many types of pages.

**Comment by MikeRosTX** [comment]
> > in the REFERENCE.md under the Speaker_page_fields.
> 
> Huh?

I thought the field was speaker specific, but you mentioned in a previous comment that it's a global change. So my comment is null. 😉

**Comment by adrianmoisey** [comment]
I've added documentation. Not sure it's the best, so suggestions are welcome.

If everyone is happy, I'll most likely close this PR and open another one with a cleaner git history (and no examples)

**Comment by mattstratton** [comment]
That sounds good, @adrianmoisey 	- go ahead and close this one and open a cleaner PR. Thanks!!!

**Comment by adrianmoisey** [comment]
Done.

**Pull request closed without merging**